contributing.md

```diff
@@ -31,14 +31,68 @@ If you're new to open source and don't know where to start, don't worry! We've t
 **Prerequisites**
```

Comment:
Make Prerequisites in new Line

Reply:
Could you kindly provide all the changes you'd like in one go, please